### PR TITLE
initial TheKey configuration & logout support

### PR DESCRIPTION
**Pull request by frett**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -10,6 +10,8 @@ android {
         targetSdkVersion 19
         versionCode 1
         versionName "1.0"
+
+        buildConfigField "long", "THEKEY_CLIENTID", "7439283632057308L"
```
Comment (wrandall22) [inline app/build.gradle]: Is this safe to have in a public repo?

Comment (frett) [inline app/build.gradle]: yeah, it's not considered private information. plus anyone could extract it from the published APK anyways

**Comment by wrandall22** [comment]
Thanks for the head start on this, Daniel.

**Comment by frett** [comment]
gladly, feel free to direct questions regarding The Key OAuth library my way, I'm the one that wrote it, so I should be able to help with it :)

**Pull request closed without merging**